src/pe-sticky-header/widget/pe-sticky-header.u-css.scss

```diff
@@ -0,0 +1,162 @@
+$pe-top-navbar-background-color: #293E40 !default; 
+$pe-top-navbar-background-img: none !default;
+$pe-top-navbar-height: 70px !default;
+
+/* Sticky navbar */
+.sticky {
+  position: fixed;
+  top: 0;
+  width: 100%;
+}
+
+.sticky + .body {
+  padding-top: 60px;
+}
+
+@media screen and (max-width: 768px){
+  .top-navbar {
+    z-index: 9001;
+    top: 0px;
+  }
+
+  nav.navbar {
+    top: 0px;
+    margin-top: -60px;
+    padding-top: 60px;
+  }
+}
+
+nav {
+  //overflow: hidden;
+  z-index: $zindex-navbar;
+}
+
+.top-navbar {
+  background-color: $pe-top-navbar-background-color;
+  background-image: $pe-top-navbar-background-img;
+  height: $pe-top-navbar-height;
+
+  position: relative;
+  
+  a, button {
+    margin: 0;
+    position: absolute;
+    top: 50%;
+    transform: translate(0, -50%);
+  }
+
+  button {
+    right: 2rem;
+  }
+
+}
+
+#nav-logo {
+  margin-left: 10px;
+}
```

Comment:
add a class to id, instead of applying style to id.

`<div id="#nav-logo" class="some-class">`